lib/ProductOpener/Ingredients.pm

```diff
@@ -1976,6 +1979,98 @@ sub set_percent_sub_ingredients($) {
 }
 
 
+=head2 compute_ingredients_percent_values ( total_min, total_max, ingredients_ref )
```

Comment:
this looks like the blurb for the wrong function?

Reply:
indeed, I just changed it.